src/pages/candidate-details/candidate-details.ts

```diff
@@ -50,13 +56,29 @@ export class CandidateDetailsPage extends BasePageComponent implements OnInit, O
   ) {
     super(platform, navController, authenticationProvider);
     this.slotId = navParams.get('slotId');
+    this.slotChanged = navParams.get('slotChanged');
   }
 
   ngOnInit(): void {
-    
-    this.store$.dispatch(new ClearChangedSlot(this.slotId));
-
     this.pageState = {
+      specialNeeds$: this.store$.pipe(
+        select(getJournalState),
+        select(getSlots),
+        map(slots=>getSlotById(slots, this.slotId)),
```

Comment:
Arrow function style here isn't consistent with the rest of the codebase.

`argument => body()`